x-pack/plugins/cases/server/client/configure/validators.ts

```diff
@@ -48,6 +50,38 @@ export const validateCustomFieldTypesInRequest = ({
   }
 };
 
+export const validateCustomFieldDefaultValuesInRequest = ({
```

Comment:
nit:

```suggestion
export const validateListCustomFieldDefaultValuesInRequest = ({
```

Reply:
I'd rather give this a generic name like everything else called in `x-pack/plugins/cases/server/client/configure/client.ts` in case we later add new custom field types which need their default values validated.